extensions/protocols/dcp/dcp-issuer/dcp-issuer-api/src/main/java/org/eclipse/edc/identityhub/protocols/dcp/issuer/api/v1alpha/ApiSchema.java

```diff
@@ -0,0 +1,182 @@
+    ) {
+        public static final String API_ERROR_EXAMPLE = """
+                {
+                    "message": "error message",
+                    "type": "ErrorType",
+                    "path": "object.error.path",
+                    "invalidValue": "this value is not valid"
+                }
+                """;
+    }
+
+    @Schema(name = "CredentialRequestMessage", example = CredentialRequestMessageSchema.RESPONSE_EXAMPLE)
+    record CredentialRequestMessageSchema(
+            @Schema(name = CONTEXT, requiredMode = REQUIRED)
+            Object context,
+            @Schema(requiredMode = REQUIRED)
+            List<CredentialRequestSchema> credentials
+    ) {
+
+        public static final String RESPONSE_EXAMPLE = """
+                {
+                   "@context": [
+                     "https://w3id.org/dspace-dcp/v1.0/dcp.jsonld"
+                   ],
+                   "type": "CredentialRequestMessage",
+                   "credentials": [
+                     {
+                       "credentialType": "MembershipCredential",
+                       "format": "vcdm11_jwt"
+                     },
+                     {
+                       "credentialType": "OrganizationCredential",
+                       "format": "vcdm11_ld"
+                     },
+                     {
+                       "credentialType": "Iso9001Credential",
+                       "format": "vcdm20_jose"
+                     }
+                   ]
+                 }
+                """;
+    }
+
+    @Schema(name = "CredentialRequest", example = CredentialRequestSchema.EXAMPLE)
+    record CredentialRequestSchema(
+            @Schema(name = "credentialType", requiredMode = REQUIRED)
+            String credentialType,
+            @Schema(name = "format", requiredMode = REQUIRED)
+            String format
+    ) {
+        public static final String EXAMPLE = """
+                {
+                       "credentialType": "MembershipCredential",
+                       "format": "vcdm11_jwt"
+                }
+                """;
+    }
+
+    @Schema(name = "CredentialStatus", example = CredentialStatusSchema.RESPONSE_EXAMPLE)
+    record CredentialStatusSchema(
+            @Schema(name = CONTEXT, requiredMode = REQUIRED)
+            Object context,
+            @Schema(name = "type", requiredMode = REQUIRED)
+            String type,
+            @Schema(name = "status", requiredMode = REQUIRED)
+            String status
+    ) {
+
+        public static final String RESPONSE_EXAMPLE = """
+                {
+                   "@context": [
+                     "https://w3id.org/dspace-dcp/v1.0/dcp.jsonld"
+                   ],
+                   "type": "CredentialStatus",
+                   "requestId": "requestId",
+                   "status": "RECEIVED"
+                 }
+                """;
+    }
+
+    @Schema(name = "IssuerMetadata", example = IssuerMetadataSchema.RESPONSE_EXAMPLE)
```

Comment:
I wonder if we should utilize the `externalDoc` functionality of OpenAPI to directly reference relevant parts of the DCP specification...
for a separate PR of course, but that would guarantee consistency.

Reply:
We could also remove the `OpenApi` docs and use directly the specification as we do for `DSP`

Reply:
i'm OK with both